sovtokenfees/doc/Interface/README.md

```diff
@@ -377,3 +377,71 @@ Example:
     }
 }
```

Comment:
Please also add docs for default auth rules for XFER, MINT, SET_FEES (create `auth_rules.md` mentioning that it extends the auth rules from Indy Node).